helper/calculateNetworth.js

```diff
@@ -3,80 +3,107 @@ const { calculateSackItem } = require('../calculators/sacksCalculator');
 const { calculateEssence } = require('../calculators/essenceCalculator');
 const { calculateItem } = require('../calculators/itemCalculator');
 const { getPetLevel } = require('../constants/pets');
+const { NetworthTypeError } = require('./errors');
+const NetworthTypes = require('../NetworthTypes');
 
-const calculateNetworth = (items, purseBalance, bankBalance, personalBankBalance, prices, onlyNetworth, returnItemData) => {
-  const categories = {};
+const calculateNetworth = (items, purseBalance, bankBalance, personalBankBalance, prices, options) => {
+    const { networthType, onlyNetworth, includeItemData, stackItems } = options;
+    validateTypes(networthType);
```

Comment:
This line should move to the `NetworthManager` constructor and setter for `networthType`.

Reply:
good point